packages/api/src/router/appSessions.ts

```diff
@@ -223,6 +182,79 @@ export const appSessionsRouter = router({
     const remaining = await rateLimits.appSessions.demo.getRemaining(userId);
     return { remaining };
   }),
+  modifySection: protectedProcedure
```

Comment:
I think that these might belong in their own router. Ultimately, all functionality we're building relates to app sessions in some way, but this could be extracted to be more specific to feedback

Reply:
Yep, def agree

Reply:
Because I don't have time I'm going to say I'll ticket this for post launch.

Reply:
I did actually make a chat router at one point but it fell foul of a merge with another big PR